src/components/Home.jsx

```diff
@@ -0,0 +1,143 @@
+        const { balance, filteredTransactions, stringFilters } = state
+        const { deposites, withdrawels } = this.getTransactions()
+        const { categories, vendors } = stringFilters
+        return <div className="home">
+            <Filters setDateFilters={setDateFilters} categories={categories} vendors={vendors} setFilters={setFilters}/>
+            <h1 className="title">Transactions</h1>
+            <div className="transactionsContainer">
+                <div className="accountStatus">
+                    <div className="balanceContainer">
+                        <span> Balance: </span>
+                        <span className={`balance ${balance < 0 ? "redTxt" : "greenTxt"}`}> {balance} ₪</span>
+                    </div>
+                    <div className="balanceContainer Withdrawels">
+                        <span> Withdrawels: </span>
+                        <span className={`balance redTxt `}> {withdrawels} ₪</span>
+                    </div>
+                    <div className="balanceContainer">
+                        <span> Deposits: </span>
+                        <span className={`balance greenTxt`}> {deposites} ₪</span>
+                    </div>
+                </div>
+                <div className="headers">
+                    <div title="click to sort" className="property" onClick={() => sort("date")}>
+                        Date  {this.sortingIcon("date")}
+                    </div>
+                    <div title="click to sort" className="property" onClick={() => sort("vendor")}>
+                        Vendor  {this.sortingIcon("vendor")}
+                    </div>
+                    <div title="click to sort" className="property" onClick={() => sort("category")}>
+                        Category  {this.sortingIcon("category")}
+                    </div>
+                    <div title="click to sort" className="property " onClick={() => sort("amount")}>
+                        Amount  {this.sortingIcon("amount")}
+                    </div>
+                    <div className="tableBtns">
+                        <Link to="/operation" className="newTransactionBtn" title="Add transaction">
+                            <FontAwesomeIcon icon={faPlusCircle} />
+                        </Link>
+                        <FontAwesomeIcon icon={faSync} className="resetFiltersHome" onClick={this.resetFilters} />
+                    </div>
+                </div>
+                <div className="filtertransactionsContainer">
+
+                {!(filteredTransactions.length === 0) ?
+                    filteredTransactions.map(t => (categories[t.category] || vendors[t.vendor]) && <Transaction key={t._id} transaction={t} deleteTransaction={deleteTransaction} />) :
+                    <div className="nothingTOSshow"> No transactions to show</div>
+                }
+                </div>
+            </div>
+        </div>
+    }
```

Comment:
better to split all the JSX to different smaller components, makes it more readable and also that's the React way of doing things (using small components)